src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -241,14 +241,16 @@ class ContentOpenWith extends PureComponent<Props, State> {
      * @private
      * @return {void}
      */
-    onIntegrationClick = (appIntegrationId: string): void => {
+    onIntegrationClick = (displayIntegration: Integration): void => {
```

Comment:
You can destructure this in the function signature, if you want:

`onIntegrationClick = ({ appIntegrationId, displayName }: Integration): void => {`